### DISCUSSION TITLE
8VAC Transformers?

### DISCUSSION THREAD

**Discussion by jhogendorn** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#100] [General]
Hi, the docs state a 110/240VAC transformer to 9/12VAC output requirement.

Is it possible to use/recalibrate to a 8VAC supply?

I ask because 'bell transformers' in din rail packages are common and much, much, much cheaper. ie:

~$30 https://au.element14.com/abb/ts8-8sw/bell-transformer-8v-1a-8va/dp/3293975
vs
~$60 https://au.element14.com/abb/tm10-12/bell-transformer-4v-8v-12v-2-5a/dp/3293953

**Comment by CircuitSetup** [comment]
Yes, the 8VAC one will work, I'll just have to be calibrated to the meter.

**Reply by jhogendorn** [reply]
Thanks so much :)
